airgun/views/syncplan.py

```diff
@@ -0,0 +1,70 @@
+from widgetastic.widget import Select, Text, TextInput, View
+
+from airgun.views.common import (
+    AddRemoveResourcesView,
+    BaseLoggedInView,
+    SatTab,
+    SearchableViewMixin,
+)
+from airgun.widgets import (
+    ConfirmationDialog,
+    DateTime,
+    EditableEntry,
+    EditableEntryCheckbox,
+    EditableDateTime,
+    EditableEntrySelect,
+    ReadOnlyEntry,
+    SelectActionList,
+)
+
+
+class SyncPlanView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h2[contains(., 'Sync Plans')]")
+    new = Text("//button[contains(@href, '/sync_plans/new')]")
+    edit = Text(
+        "//td/a[contains(@ui-sref, 'sync-plan') and "
+        "contains(@href, 'sync_plans')]"
+    )
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(
+            self.title, exception=False) is not None
+
+
+class SyncPlanCreateView(BaseLoggedInView):
+    name = TextInput(id='name')
+    description = TextInput(id='description')
+    interval = Select(id='interval')
+    date_time = DateTime()
+    submit = Text("//button[contains(@ng-click, 'handleSave')]")
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(
+            self.name, exception=False) is not None
+
+
+class SyncPlanEditView(BaseLoggedInView):
+    return_to_all = Text("//a[text()='Sync Plans']")
```

Comment:
Please leave some kind of `fixme` or `todo` comment - that's not separate link, that's patternfly's `Breadcrumb` widget which we should implement and remove this `Text` widget afterwards